src/app/locale/translations/en.json

```diff
@@ -258,6 +258,9 @@
             "paypal_continue_action": "Continue with PayPal",
             "braintreevenmo_continue_action": "Continue with Venmo",
             "paypal_credit_continue_action": "Continue with PayPal Credit",
+            "pay_later_continue_action": "Continue with Pay Later",
+            "paypal_ratenzahlung_continue_action": "Continue with Paypal Ratenzahlung",
+            "pay_in_4_continue_action": "Continue with Pay in 4",
```

Comment:
why do we need those translations? We've received from BE already translated values

Reply:
Ok, i will remove